### DISCUSSION TITLE
BUILD SERVER - THE SUGGESTION THAT CANT WAIT

### DISCUSSION THREAD

**Discussion by d4v3cc** [TAServers/issues#28] [Ideas]
**ITS THE MOST IMPORTANT CHANGE!!!!!!!!!!**

**Comment by d4v3cc** [comment]
FEATURES - 

- BUILDING TOOLS THAT WILL BE LATER LISTED
- NO XANDER (makes fucking sense obviously)
- NO STARFALL/HIGHLY RESTRICTED
- NO XANDER STUFF RELATED (GAD, i dont want to risk getting my whole fucking cp folder getting logged onto HIS CP folder)
- NO ADMIN RANKS (for what reason right?)

**Reply by Derpius** [reply]
Xander would only be banned if he crashes

starfall will be added and unrestricted (used by many builders and you haven’t given a reason to not add it)

if/when GAD is ready we’ll be rolling it out to all servers

no admin ranks is fine

**Reply by d4v3cc** [reply]
today he literally propblocked everyone while we were building

**Reply by d4v3cc** [reply]
literally for no reason

**Reply by Xandertron** [reply]
just the JL members :)

**Reply by d4v3cc** [reply]
kek since when is sethler in JL

**Reply by Derpius** [reply]
> today he literally propblocked everyone while we were building

Well other people are also capable of doing that, so you need to come up with a solution that applies to everyone (a harsh one being disabling propcore)
